instrumentation/opentelemetry-api/opentelemetry-api-1.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/opentelemetryapi/OpenTelemetryInstrumentation.java

```diff
@@ -47,4 +61,18 @@ public static void methodExit(
       openTelemetry = ApplicationOpenTelemetry.INSTANCE;
     }
   }
+
+  @SuppressWarnings("unused")
+  public static class SetAdvice {
+
+    @Advice.OnMethodEnter(suppress = Throwable.class)
+    public static void onEnter() {
+      LoggerFactory.getLogger(application.io.opentelemetry.api.GlobalOpenTelemetry.class)
+          .warn(
```

Comment:
maybe info? (which is the javaagent's default log level, so will still get logged)

Reply:
I think that `warn` makes slightly more sense in this case (since the log _warns_ you about incorrect API usage), but I don't really have a strong opinion on this. I'll change that if we decide so.

Reply:
my only concern is about users adding the javaagent later to an existing app that was previously manually instrumented, and then they're stuck with this warning message which is not really actionable

let's keep at `warn` and we can always revisit later

Reply:
> my only concern is about users adding the javaagent later to an existing app that was previously manually instrumented, and then they're stuck with this warning message which is not really actionable
> 
> let's keep at `warn` and we can always revisit later

Why not actionable? They can remove their manual usages of `GlobalOpenTelemetry.set`?

Reply:
sorry, I should have been more specific, I was thinking about ops users